turf/src/commonMain/kotlin/io/github/elcolto/geokjson/turf/booleans/Within.kt

```diff
@@ -0,0 +1,226 @@
+@file:Suppress("TooManyFunctions")
+
+package io.github.elcolto.geokjson.turf.booleans
+
+import io.github.elcolto.geokjson.geojson.BoundingBox
+import io.github.elcolto.geokjson.geojson.Geometry
+import io.github.elcolto.geokjson.geojson.GeometryCollection
+import io.github.elcolto.geokjson.geojson.LineString
+import io.github.elcolto.geokjson.geojson.MultiLineString
+import io.github.elcolto.geokjson.geojson.MultiPoint
+import io.github.elcolto.geokjson.geojson.MultiPolygon
+import io.github.elcolto.geokjson.geojson.Point
+import io.github.elcolto.geokjson.geojson.Polygon
+import io.github.elcolto.geokjson.geojson.Position
+import io.github.elcolto.geokjson.turf.ExperimentalTurfApi
+import io.github.elcolto.geokjson.turf.coordAll
+import io.github.elcolto.geokjson.turf.measurement.computeBbox
+import io.github.elcolto.geokjson.turf.measurement.midpoint
+
+/**
+ * @return true if [geometry1] is within [geometry2].
+ *
+ * @throws IllegalStateException when the geometries are not comparable to each other.
+ */
+@Suppress("CyclomaticComplexMethod")
+@ExperimentalTurfApi
+@Throws(IllegalStateException::class)
```

Comment:
Extend documentation, by detailed information, similar  to [turfjs doc](https://turfjs.org/docs/api/booleanWithin)

```
Boolean-within returns true if the first geometry is completely within the second geometry. The interiors of both geometries must intersect and, the interior and boundary of the primary (geometry a) must not intersect the exterior of the secondary (geometry b). Boolean-within returns the exact opposite result of the @turf/boolean-contains.
```

Reply:
Changed the doc.